WordPress/Classes/Utility/Networking/RequestAuthenticator.swift

```diff
@@ -182,6 +191,36 @@ class RequestAuthenticator: NSObject {
         }
     }
 
+    private func requestForAtomicWPCom(url: URL, loginURL: String, cookieJar: CookieJar, username: String, authToken: String, completion: @escaping (URLRequest) -> Void) {
+
+        func done() {
+            // For non-private Atomic sites, proxy the request through wp-login like Calypso does.
+            // If the site has SSO enabled auth should happen and we get redirected to our preview.
+            // If SSO is not enabled wp-admin prompts for credentials, then redirected.
+            var components = URLComponents(string: loginURL)
+            var queryItems = components?.queryItems ?? []
+            queryItems.append(URLQueryItem(name: "redirect_to", value: url.absoluteString))
+            components?.queryItems = queryItems
+            let requestURL = components?.url ?? url
+
+            let request = URLRequest(url: requestURL)
+            completion(request)
+        }
+
+        authenticationService.loadAuthCookiesForWPCom(into: cookieJar, username: username, authToken: authToken, success: {
+            done()
```

Comment:
Fairly minor, but this jumped out at me while looking at the code.  I think you can simplify the success param to just be:

```swift
success: done,
```

(Without the `{}`s and `()`s)

Reply:
True. If you don't mind I think I'll leave it as is so it's consistent with the other methods in that class.   Thanks for taking a peek!